ace.d.ts

```diff
@@ -371,6 +371,7 @@ export namespace Ace {
     all(): { [key: string]: any };
     moduleUrl(name: string, component?: string): string;
     setModuleUrl(name: string, subst: string): string;
+    setModuleLoader(name: string, onLoad: Function): void;
```

Comment:
looks like setLoader is missing here too

Reply:
Added